packages/sdk-webos/src/deviceManager.ts

```diff
@@ -123,8 +125,23 @@ const parseDevices = (c: RnvContext, devicesResponse: string): Promise<Array<Web
 const launchAppOnSimulator = async (c: RnvContext, appPath: string) => {
     logTask('launchAppOnSimulator');
 
+    const bundleAssets = getConfigProp(c, c.platform, 'bundleAssets');
     const webosSdkPath = getRealPath(c, c.buildConfig?.sdks?.WEBOS_SDK);
 
+    const appLocation = bundleAssets ? path.join(appPath, 'build') : appPath;
+
+    if (bundleAssets) {
```

Comment:
@mihaiblaga89  `deviceManager` should not be responsible for any build related activities or decisions.  `buildCoreWebpackProject` more logical place to incorporate such logic

Reply:
I do agree on deviceManager but I do not agree with buildCoreWebpackProject. Webpack sdk should not be aware of webos specific files. Moved the copy to `runWebOS`. Also fixed release scheme clean build (it would fail if you would pass `-r`)